mleap-serving/src/main/resources/application.conf

```diff
@@ -1,8 +1,6 @@
 ml.combust.mleap.serving {
   http {
-    hostname = 0.0.0.0
-    port = 65327
-    bind-hostname = ${ml.combust.mleap.serving.http.hostname}
-    bind-port = ${ml.combust.mleap.serving.http.port}
+    bind-hostname = 0.0.0.0
```

Comment:
We can rewrite this line as:
```
bind-hostname = 0.0.0.0
basedir = ${?MLEAP_SERVER_HOSTNAME}
```

This will interpolate the environment variable

Reply:
Ok! Done. Thank you, too, for the additional context on the future state of the MLeap server infrastructure.  We're looking forward to that, but this in the meantime will really help us!